Updated world.json

Rydela opened the pull request:
**Change log:**
15/05/2020

Updated the world.json file, deleting unused fields and adding ISO2 codes. Fixes issue #57.

**Field deleted:**

ID
Soveriegnt
Type
Name_Long
Abbrev
Formal_EN
Post_est
LastCensus
Continent
Zika
Zika_level
Zika_Cases
Alert
Case_source
Postal

**Updated:**

France ISO3 Code
Kosovo ISO3 Code
Norway ISO3 Code

**Added:**

ISO2 codes to all countries in dataset

**Fields remaining:**

Name
ISO3 (renamed)
ISO2 (added)

Rydela committed:
Updated world.json

Change log:
15/05/2020

Updated the world.json file, deleting unused fields and adding ISO2 codes.

Field deleted:

ID
Soveriegnt
Type
Name_Long
Abbrev
Formal_EN
Post_est
LastCensus
Continent
Zika
Zika_level
Zika_Cases
Alert
Case_source
Postal

Fields remaining:

Name
ISO3 (renamed)
ISO2 (added)

Updated:
France ISO3 Code
Kosovo ISO3 Code
Norway ISO3 Code

Added:

ISO2 codes to all countries in dataset

SimonbJohnson commented:
Not merging this yet as it will break the ISO3 mapping.

The 'name' variable on this line needs to be updated for the new field name in the geom file.

https://github.com/SimonbJohnson/quickX3/blob/master/hxldash/static/libs/hxlbites/hxlBitesMap.js#L59

Rydela committed:
Changed ISO3 name in world.json

Changed "ISO_A3" to "ISO3" in order to visualize world.json file.

Rydela committed:
Added ISO2 to geom file.

Cleaned up world.json file. Should be much easier to look at now.

This also solves issue #79 . It is possible to create dashboards using ISO2 codes.

SimonbJohnson committed:
allowing lowercase iso codes to be used
